packages/webassemblyjs/test/index.js

```diff
@@ -46,10 +47,15 @@ describe("interpreter", () => {
   });
 
   describe("wat", () => {
-    const testSuites = glob.sync(
+    let testSuites = glob.sync(
       "packages/webassemblyjs/test/fixtures/**/module.wast"
     );
 
+    // Filter based on last command line argument
+    if (argv.only) {
+      testSuites = testSuites.filter(f => f.includes(argv.only));
+    }
```

Comment:
I added a way to select tests, see [You can select test based on their titles](https://github.com/xtuc/webassemblyjs/blob/master/CONTRIBUTING.md#running-lintingtests). Would that fix this?

Reply:
Ahh maybe I should have listened to GitHub when it told me "CONTRIBUTING.md has changed..." :grin: Thanks!